src/components/connect.tsx

```diff
@@ -465,18 +465,18 @@ function connect<
 
   const Context = context
 
-  type WrappedComponentProps = TOwnProps & ConnectProps
-
   const initMapStateToProps = mapStateToPropsFactory(mapStateToProps)
   const initMapDispatchToProps = mapDispatchToPropsFactory(mapDispatchToProps)
   const initMergeProps = mergePropsFactory(mergeProps)
 
   const shouldHandleStateChanges = Boolean(mapStateToProps)
 
-  const wrapWithConnect: AdvancedComponentDecorator<
-    TOwnProps,
-    WrappedComponentProps
-  > = (WrappedComponent) => {
+  const wrapWithConnect = <TProps,>(
```

Comment:
is the comma a typo?

Reply:
No, it was actually necessary!

When I removed it, TS thought that `<TProps>` was the start of a JSX element, because this _is_ a `.tsx` file.  I had to leave it in to convince TS "nope, this is a generic type declaration" (presumably similar to how in Python `(x)` is just parentheses, and `(x,)` is a tuple).